posthog/migrations/0272_alter_organization_plugins_access_level.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 3.2.15 on 2022-10-18 06:42
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('posthog', '0271_delete_promptsequencestate'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='organization',
+            name='plugins_access_level',
+            field=models.PositiveSmallIntegerField(choices=[(0, 'none'), (3, 'config'), (6, 'install'), (9, 'root')], default=3),
```

Comment:
This should be a safe change as this was anyway the default for cloud and for self-hosted this is only affects new installations